public/js/core.js

```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Enter your key!",
-			text: "Enter the key provided by the bot:",
+			html: true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link: ' +
+							'<a href="https://t.me/Tarchivebot" target="_blank">Tarchivebot</a><br>' +
```

Comment:
While the link is completely fine in our situation if anyone else decides to run this bot the link will point to the wrong place.

As far as I'm aware there is a variable in the backend that is exported to be used in the pug, so you could have it be accessible like this if I recall correctly:
```pug
script.
    var botLink = {accessibleVar}
```

@fuerbringer probably knows a bit more on this.